internal/converters/converters.go

```diff
@@ -14,15 +14,15 @@ func contains(s []string, e string) bool {
 	return false
 }
 
-func stringArrToIfArr(sli []string) []interface{} {
+func StringArrToIfArr(sli []string) []interface{} {
 	var arr []interface{}
 	for _, v := range sli {
 		arr = append(arr, v)
 	}
 	return arr
 }
 
-func convertStringArr(ifaceArr []interface{}) []string {
+func ConvertStringArr(ifaceArr []interface{}) []string {
```

Comment:
Let's take this opportunity to improve these function names.  `StringArrToIfArr` is a nice, descriptive name, and if we follow that model here, this function would be called `IfArrtoStringArr`